.github/workflows/perf_comp_api_tests.yaml

```diff
@@ -0,0 +1,93 @@
+env:
+  GALAXY_TEST_DBURI_TARGET: 'postgres://postgres:postgres@localhost:5432/galaxy_target?client_encoding=utf8'
+  GALAXY_TEST_DBURI_PR: 'postgres://postgres:postgres@localhost:5432/galaxy_pr?client_encoding=utf8'
+  # GALAXY_TEST_COLLECT_STATSD: 'true'
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        python-version: [3.7]
+    services:
+      # metrics:
+      #  image: samuelebistoletti/docker-statsd-influxdb-grafana:latest
+      #  ports:
+      #  - 3003:3003
+      #  - 3004:8888
+      #  - 8086:8086
+      #  - 8125:8125/udp
+      postgres:
+        image: postgres:11
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: postgres
+        ports:
+          - 5432:5432
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        path: 'galaxy root'
+    - uses: actions/checkout@v2
+      with:
+        path: 'galaxy root base'
+        ref: ${{ github.event.pull_request.head.sha }}
+    - name: Clone galaxyproject/galaxy-test-data
+      uses: actions/checkout@v2
+      with:
+        repository: galaxyproject/galaxy-test-data
+        path: galaxy-test-data
+    - uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Cache venv dir
+      uses: actions/cache@v1
+      id: pip-cache
+      with:
+        path: ~/.cache/pip
+        key: pip-cache-${{ matrix.python-version }}-${{ hashFiles('galaxy root/requirements.txt') }}
+    - name: Move test data
+      run: rsync -av --remove-source-files --exclude .git galaxy-test-data/ 'galaxy root/test-data/'
+    - name: Move test data base
+      run: rsync -av --remove-source-files --exclude .git galaxy-test-data/ 'galaxy root base/test-data/'
+    - name: Run tests on PR branch
+      run: GALAXY_TEST_DBURI=$GALAXY_TEST_DBURI_PR ./run_tests.sh --ci_test_metrics --structured_data_report_file "pr.json" -api lib/galaxy_test/performance
+      working-directory: 'galaxy root'
+    - name: Run tests on target branch
+      run: GALAXY_TEST_DBURI=$GALAXY_TEST_DBURI_TARGET ./run_tests.sh --ci_test_metrics --structured_data_report_file "target.json" -api lib/galaxy_test/performance
+      working-directory: 'galaxy root base'
+    - name: Generate test HTML
+      run: '. .venv/bin/activate; python scripts/tests_markdown.py --output_path pr.html pr.json'
+      working-directory: 'galaxy root'
+    - uses: actions/upload-artifact@v2.0.1
+      with:
+        name: 'Performance Metrics PR'
+        path: 'galaxy root/pr.html'
+    - name: Generate combined test HTML
+      run: '. .venv/bin/activate; python scripts/tests_markdown.py --output_path pr_vs_target.html pr.json "../galaxy root base/target.json"'
+      working-directory: 'galaxy root'
+    - uses: actions/upload-artifact@v2.0.1
+      with:
+        name: 'Performance Metrics (PR vs Target)'
+        path: 'galaxy root/pr_vs_target.html'
+    #- name: Sleep 5
```

Comment:
I'm fine with this block (and the block above) being commented out; but just checking - is it supposed to be left in the final version? And if so, would you mind adding a brief comment explaining its purpose? (I think it would be helpful, unless it's completely obvious, in which case please ignore my comment)

Reply:
I think I meant to drop this whole file when cleaning it up - I wanted to find a way to just do this on request. I'll rebase without this.